docs/UserGuide.md

```diff
@@ -21,9 +21,9 @@ Click on each command to jump to their subsection.
 | **[Add](#adding-a-person--add)**                  | `add n/NAME i/STUDENT_ID p/PHONE e/EMAIL [t/TAG]…​` <br> e.g., `add n/James Ho i/E0000001 p/22224444 e/jamesho@example.com t/friend t/colleague` |
 | **[List](#listing-all-persons--list)**            | `list`                                                                                                                                           |
 | **[Edit](#editing-a-person--edit)**               | `edit INDEX [n/NAME] [i/STUDENT_ID] [p/PHONE] [e/EMAIL] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`                          |
-| **[Mark](#marking-person-as-attended--mark)**     | `mark INDEX tut/TUTORIAL`<br> e.g., `mark 2 tut/1`                                                                                               |
-| **[Unmark](#marking-person-as-absent--unmark)**   | `unmark INDEX tut/TUTORIAL`<br> e.g., `unmark 2 tut/1`                                                                                           |
-| **[Reset](#resetting-persons-attendance--reset)** | `reset INDEX tut/TUTORIAL`<br> e.g., `reset 2 tut/1`                                                                                             |
+| **[Mark](#marking-person-as-attended--mark)**     | `mark INDEX tut/TUTORIAL` <br>`INDEX` can be single, range (e.g., `1 - 12`), or `*` for all <br> e.g., `mark 2 tut/1`                            |
```

Comment:
Shouldn't this be `TUTORIAL` can be single, range... and not `INDEX`? Similar for the next two.

Perhaps you could also talk about how `INDEX` can be a wildcard?